### PR TITLE
[skipci] doc: add space manage of curvefs with volume backend

### PR DESCRIPTION
**Pull request by ilixiaocui**
### What problem does this PR solve?

Issue Number: #2160  

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?):

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

File: docs/cn/Curve文件系统Volume后端空间管理.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+**解决方法：** 数据清理模块持久化需要处理的InodeId
+
+具体说明：在 Inode 的 nlink 被更新之前加入到数据清理模块，数据清理模块将对应的 InodeId 持久化。异常情况下 nlink 未更新，InodeId 已被持久化，清理模块在扫描的时候判断一下即可。
+
+优点：
+- 不需要全部遍历所有的 InodeId, 清理效率会大大提高
+
+缺点：
+- 三副本上需要同步这一信息，因此 InodeId 的持久化需要通过一致性协议，在删除的文件数量很大的情况下，会有比较大的时间开销
```
Comment (cw123) [inline docs/cn/Curve文件系统Volume后端空间管理.md:143]: 这个持久化，可以在update inode的时候做，不需要额外的走一次一致性协议，时间开销上会增加一次持久化rocksdb的操作，额外增加的时间开销应该还好。

File: docs/cn/Curve文件系统Volume后端空间管理.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+Curve文件系统中:
+- **文件的写入需要分配空间**，因此 ```Client``` 需要对 ```BlockGroup``` 进行操作以分配空间。
+- **文件的删除需要清理空间**，元数据是在 ```Metaserver``` 管理的，因此 ```Metaserver``` 需要对 ```BlockGroup``` 进行操作以回收空间。
+
+当前空间回收流程如下：
+
+1. ```Client``` 收到用户删除文件的请求 （空间回收的触发条件1）并处理
+
+    - 如果配置了回收站，则把文件放到回收站中（相当于rename的操作)
+    - 如果没有配置回收站，则直接调用删除Inode的请求
```
Comment (cw123) [inline docs/cn/Curve文件系统Volume后端空间管理.md:46]: client这边是update inode

Comment (ilixiaocui) [reply docs/cn/Curve文件系统Volume后端空间管理.md:46]: 是的

File: docs/cn/Curve文件系统Volume后端空间管理.md (hunk 1 above)
Comment (ilixiaocui) [reply docs/cn/Curve文件系统Volume后端空间管理.md:143]: 这里不是指删除的时候，是指在数据清理模块，把VolumeExtent对应的BlockGroup的bitmap清理掉后，inode需要更新。

**Pull request merged**